stable/datadog/values.yaml

```diff
@@ -360,6 +360,7 @@ rbac:
   serviceAccountName: default
 
 tolerations: []
+podLabels: {}
```

Comment:
Could you move it under the `daemonset` section please? We're in the process of deprecating the deployment, in favor of `clusterchecksDeployment`, so it's on feature freeze.

Reply:
Done